app/protractor.conf.js

```diff
@@ -0,0 +1,28 @@
+exports.config = {
+
+    allScriptsTimeout: 11000,
+
+//   specs: [
+//     'test-e2e/**/*.js'
+//   ],
+
+    suites: {
+        home:  'test-e2e/home.spec.js',
+        //full: 'test-e2e/**/*.spec.js',
+        mytest: 'test-e2e/myapp.spec.js'
+    },
+
+    capabilities: {
+        'browserName': 'chrome'
+    },
+
+    seleniumAddress: 'http://localhost:4444/wd/hub',
```

Comment:
If you leave this line in, you need to manually start the selienum server. If you leave it out, tests will auto-start an instance.